docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+          
+    <remembered-set>
+
+    </mem-info>
+
+<gc-end>
+
+<concurrent-start type="GMP work packet processing"/> (1st GMP concurrent subincrement starts)
+
+<exclusive-end/>                                      (STW pause ends)
+
+<concurrent-end type="GMP work packet processing"/>   (1st GMP concurrent subincrement ends)
+
+<gc-op type="mark increment"/>                        (copy forward operation runs concurrently)
+
+</concurrent-end type="GMP work packet processing"/>>
+
+...                                       (partial GC cycles run during STW pauses)
+
+<exclusive-start/>                        (STW pause starts)
+
+<gc-start type="global mark phase"/>                  (2nd STW GMP subincrement starts)
+
+...   
+
+<concurrent-start type="GMP work packet processing"/> (2nd concurrent GMP subincrement starts)
+
+...
+
+<exclusive-end/>
+
+...                                       (more partial GC cycles may run)
+
+<concurrent-end type="GMP work packet processing" />  (2nd concurrent GMP subincrement ends)
+...
+
+</concurrent-end>
+
+...                                       (more partial cycles and GMP increments interleave)
+
+<exclusive-start/>                        (STW pause starts)
+
+...
+
+<gc-start type="global mark phase"/>      (final STW GMP subincrement starts.)
+
+...   
+<gc-end type="global mark phase"/>        (final STW GMP subincrement ends. No concurrent subincrement runs)
+
+<cycle-end type="global mark phase"/>     (end of global mark cycle)
+
+<exclusive-end/>                          (STW pause ends)    
+
+<exclusive-start/>                        (STW pause starts)
+
+<cycle-start type="partial gc" />         (partial cycle starts)
+
+...
+
+<gc-op type="sweep" />                    (Sweep operation associated with global mark cycle runs)
+
+...
+
+<cycle-end type="partial gc"/>            (partial GC cycle ends)
+
+<exclusive-end/>                          (STw pause ends)
+
+```
+
+
+#### Global mark phase
+
+The first activity of the global mark cycle is a STW pause, recorded by an `<exclusive-start>` element. The GC pauses application threads to execute the cycle's initial operations.
+
+```xml
+<exclusive-start id="1152" timestamp="2021-02-26T11:17:25.033" intervalms="1931.263">
+  <response-info timems="3.082" idlems="1.041" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+```
+
+The cycle is triggered when the allocation taxation threshold is reached. The <allocation-taxation> element records the memory threshold value that was in use when this example global mark cycle was triggered. The`taxation-threshold` attribute shows the taxation threshold is 1024MB (1,073,741,824), which is half the total memory of the *eden* space (2048 MB), as expected for threshold triggers of global mark cycles and increments. For more information taxation thresholds for the `balanced` policy, see [`balanced` examples](vgclog_examples.md#balanced-examples).
```

Comment:
For the cycle trigger (not for the other GMP increment triggers) this a one more detail to add:

The cycle is triggered when the allocation taxation threshold is reached (half way between two PGCs) and internal heuristics determine that we are low in total free memory in heap.

Reply:
Questions:

1.Are the TWO trigger conditions, of not just the allocation taxation threshold being reached but ALSO that the GC determining the total free memory in the heap to be too low,  JUST the condition for a GLOBAL cycle? ie. if only the 1st condition is met, a pgc or global mark cycle is triggered?

2. 'allocation taxation threshold is reached halfway between 2 pgcs' - can you provide further clarification? Are we talking about a condition where 1 pgc has finished, but the next pgc has not been triggered?

Reply:
Introduction to balanced cycles has been modified to explain the taxation threshold and references to allocation taxation threshold within examples has been checked to ensure no inconsistencies or inaccuracies:

_When memory in the Java&trade; object heap reaches a memory threshold called an allocation taxation threshold, a `balanced` partial GC cycle, `balanced` global mark cycle, or `balanced` global mark cycle increment, is triggered. If available memory in the heap is low, the GC triggers a `balanaced` global mark cycle, or global mark cycle increment if the global mark cycle is in progress. Else, the GC triggers a partial cycle.

Partial GC cycles, global mark cycles, and global cycles set the allocation taxation threshold at the end of their cycle or increment to schedule the next cycle or increment. For `balanced` cycles, the *taxation* on the mutator threads refers to pausing the mutator threads while GC work is run.

When a partial cycle ends, if the cycle is not run between global mark phase increments of a global *mark* cycle, and a global mark cycle is not scheduled as the next cycle, the allocation taxation threshold is set to trigger the next cycle when the *eden* space is full. Specifically, the allocation threshold is set to be equal to the size of the *eden* space.

If a partial cycle runs within a global mark cycle, or if a global mark cycle is scheduled as the next cycle, the allocation taxation threshold, set at the end of the partial cycle, is set to be smaller than the size of the eden space.  Specifically, the allocation taxation threshold is set to be half the size of the eden space so that the next global mark cycle or global mark cycle increment triggered by the allocation threshold has enough memory available in the eden space to run._

Reply:
Also ensured that terminology is consistent - I refer to a 'taxation allocation threshold' (as opposed to 'taxation allocation' or 'taxation threshold'.